task3/vector.js

```diff
@@ -0,0 +1,81 @@
+function sortAsc(array) {
+    return array.sort((a,b) => a-b);
+}
```

Comment:
You could put this function inside Class with underscore like:
```
_sortAsc(array) {
    return array.sort((a,b) => a-b);
}
```
It will mean that _sortAsc is a 'private' method